tests/cpp/c_api/test_c_api.cc

```diff
@@ -128,15 +128,15 @@ TEST(CAPI, JsonModelIO) {
   learner->UpdateOneIter(0, p_dmat);
   BoosterHandle handle = learner.get();
 
-  std::string modelfile_0 = tempdir.path + "/model_0.json";
+  std::string modelfile_0 = tempdir.path_ + "/model_0.json";
```

Comment:
Let's not rename the member fields here, as it's outside the scope of this pull request (reduce compiler warning).